publish.gradle

```diff
@@ -18,7 +18,7 @@ publishing {
         }
     }
     tasks.withType(Sign) {
-        onlyIf { isReleaseVersion }
+        onlyIf { isReleaseVersion && System.getenv("NO_GPG_SIGN") != "true" }
```

Comment:
-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: PGP Everywhere for iOS
Comment: http://pgpeverywhere.com

xsFNBGLauzwBEADJXMBMbVozgPCtcIKIIOY++4MwK1G2sNS9JJQWOTYrbBDjrww8MUwzskkBSFOQ
2eHuQoR/xEh6S3rd55RjiTCKZXBHLQeVzwDKa9hhq8cpGlNuSBeXuSbJJb1zJYCazJE3svA+jz+s
bOQrtege0yZ8ZgmCdgJ0NCUtOm74iqx17kfPH6+CTC948HgSGrciulfwQOb6EqrUKE18r523gnal
uWmuuvJ+bG2llbf9L/IjVU6EN7sVmccKXniIu0M3ek7jE2lkzufyRv7olRJiAp+DIuOuGYn9pyNG
AZoXrosNyQ9hS5LuHjlpVmMnZFRofXZZy9y6firs1eNCJrbPU+HlqGLQn9NnS8KGGv7lbgNTT5B1
tlANZnWCiJrVFQvTSYmg5NXXDB0e7d4PfcnxeY8fWuilmkBc5uQfTulSw/mzgL1+MH6kd6COPxMP
s8mLUg42rppQCW46ajeGmheSxlmF/W7mdNdoNS2wp4dFemarsPbj/Ue9O4J8c0CTNB76VoJnKymB
1Z/5b+gmNzWOZR7HOyyMaWYEMfnoHecWRDnknudZGPMrYgmN3sHaP5dqVls2KEN/oYrWnhYK1XrD
pnLySiT0G/LI+gMpXCAnSwiNG8ga0JzNr9u8r796AY3A9mlilq09pshj7uh0jFCLBNlWSMZ53tXT
/X8Nj346P9MPrQARAQABzR1LcmFuaWV0IDxrcmFuaXVtZXRAcHJvdG9uLm1lPsLBegQTAQgAJAUC
Ytq7PAIbAwQVCAkKBwsJCAcDAgEEFgMCAQIeAQIXgAIZAQAKCRAWdA3SMwVMhmzBEADB4Otlfb4Q
173GzwbSoGZRolF28CPTxvYA70P35BQAzeRCpt4ct1M+NSyW+Z2l8PnFAMMu8YC0Z6Za71Eva15/
h+RHu16c0FnLLUAzpOtzzX5kiOcF0Ylb14n+iZ2Fge/GQfu2xUhpknpuRPAmvbjbYLXDerJM6bVE
OhI+S1HHPlj7fYRMEIRyHTVo0f2fSUV+7Gk0M8Zbn9dypkN/e6/ExTuHbdLkD1zBHYLrZ+mzjZDR
uSH1NjrfTqEdZK6rRYRjJgTsF62a5VbZUJoipLBv159bcwvytyeRK/hN+ik406spFCt5//Y2UMHv
TNGbucqwS9FNA1XogWeHAN0wXaBDIR5RKUiRJX4fx8IZtP71hFDuqY2JNn6uAm9vZZCFu9XqpUeB
sxpy+3UqWZ3+2ggVBvX6rNy//LKwHmP/B35a800tKAaLl5F4cU5eghQQiM6ZvKgvarb6cTo4qqR9
Wk/76Z1SvzteFq90NHtSoaYwiqm76XWXHmOFXnn+JIV1q1gwUJEouWFw/z8RI/BxdhoFq2ERL2o4
ZNwKINQXHIGqWQL8L421sI4l8syr/f9g6CqrsaRgUfpZ5dpffA6zBWYf+5m6DvD3ob3eKiUhthOB
WZV6ClNQmt1Jl74+gVqZ0+sjMtf2CMnJFVhnAhAvOJAYwaaQhqok5/mc/TQ3QchOhsLBbAQTAQoA
FgkQFnQN0jMFTIYFAmLa0pwFAweGH4AACgkQFnQN0jMFTIYMURAAp81VXvb0y2fq7WzkX05y36td
7n+43CFlo57i9TGBgwEFvDqS81goBZlKzGEsevEr8zMXmDO4hdsCWsT16RR+MLYEQ6xoQUUPOq/1
6qUqixLLhYXBRfJVTbdgCi4V3yr584MZj0xoNaaRbbGTB27301n4O51srUiAsUlFvvNXa8FeWVti
rfcghSgVkRWlOXc9yzHpuJZLEvchF6m0vreeTohGPiIiuiRvxB91/mfXHxViHtP9pMHQVVH4uO83
YDSDvTV3FeC3bQkTldqR7+Cy/8paAUBXJZAoytn1lvESj/JaLWWBqtCaBAgt++2p8gwWrXezuEp6
jBdXqA8P40BaUmUlWKJXp/T1QJ76kKFPzwy01swpRW5HTbtNafoXSMUuXAHZCpWDZ6gu/iSLZ01e
uY6qj1eBMpi99UNP2m9bwlGOUy05aiZOTYp8uD1BtNbGLX6AYdfOOQodrOHN9Mm8WNB/rHbSfCUl
Q5aLOKSHKSZu5qKeSV1vM5zOGcJunaH5N7hfioYN8MFpkxX6gRgNyVKmPvn+zFZgZFYXeTiSmKvl
fQLpoSalyG+ej/yMlFlTu+aClG/KD1ybPBFXfTnk643PLcfnXK1hmnyxLQycuKWqTuDPvjyJW0Jt
262dMCAsJ77bNj4a71sKZfJU8pkyxAE/1xgomULS/mbSU5Gpx3vCwWwEEgEKABYJEBZ0DdIzBUyG
BQJi2tKxBQMHhh+AAAoJEBZ0DdIzBUyGz7IP/1BAlKAIlzQmUmYacZZBucaae0RPpYgnSjyChBj7
3jo7uZefdaQLWgnNVWAGZWQBQTWNZ1wZ0RqaRyB4EYeGe+LzpjmqkiYxB5SZEL1JjHNSEoAcIPK2
IBs7E9WoEOcrkw0juZZlB+GujAbJbXLPC2j7cuiVK/o2CdvJxdYkD+0j6etREQjkfSLxTddZxqta
VQsibYXtVCHo9FZMZcohrCqTSP7LUIfg1p28JHZacYfLftgFaaF5XQMWqcrpyJLwZ38h/X2ktVOt
PmgxGHT7Tsr2XVyG6VZY/D921eD6A9+xV6zf0d6Kj0um8a26bJeup0tn1miFLmqPnQJHdFy/DId1
jWAVJBrSnvSLf6i+LxLKhWSjVfrdiRPllLL8HhzK7DR8kdyGwhDCEQcFBhILiTfvrgknni6lqbSj
VsoEtR86YKCGinn6mZAH6yz4n6AYKBMUeFdiA8gkfjHMy98pnWbxaFL67apcFlOAQmv1FMb1HGJD
toOjYb9eKgQVHOJMPQt0DX8+RFF+9GZwcbpSscJODzMqKbOfVHV1vKaHWdzonvbR4AhpRr9e+FbB
k+PvEvsyZZiYHfwKD2hhevs1EaSxgqy8MtJk3bs0EkeIb6HVd1e44p0zRhKlgfzQ4INv7uTZsgOi
K0fOH1Waa/kAHHuP3H26w1RvOnVcZkdUbWIhzsFNBGLauzwBEAClDBOmLQmbt2OTeIvWD8xsl+9Z
BGJdJoyD1Wn3EMetljs1cX9U/RY/Nr6KHLbm3TYDeVV9Iym77+DRFTtVLGtcrUzSG8OhuUgI2uH6
H+pvY0Xf3GfODaXjqL+EsTqi8WcHe/mua87oWVsE0Es77xYDRV+Giq+E7ia8cxiHJR4cy6CQudJF
hox1LRZaHoqBZ/1cN3mCZjpixROeQk/MumLWO7q4tS2YTQ3XrWkZ6J5MQ+uk5wS2MjJpdxTGMiGc
Yq76yOyjQatHQeCHxKW6lLSFGP/aPwrrophjxcbvJjiXzkeVcZuLmSMTBaiwI9C/cH2sH3FVRnMR
FlQTWj2FXS8MMKBccSOsnWEti6LQyQDtQrjeLyBLGHJ3gRdk0uYc3CKoJd3XGKaUu8Ehr6cPQZdA
N11unu4CxNxekaE4QXmFpeMeGnWVwssUE1TPbALv95haUdTsiilBtoJEbsMrUqs0t13GLQfY4WDt
5kj2RSxvZ0QDxAN6KZKHNTzwj6hQCPpJW617+mBN7540Mfo56KLdhrXW482lzYCDkRINqIq8zbcX
ow9FZFL7OA/DA/HN5c5BWC2rXI3VYZzTvLuXIhiahW58TRJZ0ygbTLHFBMELvLFI2zsZjzJd4v8L
nPgSubZKjo8prPoqef6gwrCl+rJwtaPmgPkrGxLYEC1yI1bnPQARAQABwsN+BBgBCAIoBQJi2rs/
AhsMwV0gBBkBCAAGBQJi2rs/AAoJENlf62YfbX98xQMP/iDHQw8PYt7eZyew+mxkDqkwvsBYwpie
U6hRiebDCi4Yak+IHyx9E852K4W3qwTte4RpLN7mlOzUFemxnfMZLeSlA6mvmaI+h8PTYfbO9aPR
RdsjzfDZn89NK/olrZGb/Eb3tdBNjLjQWxb485omqaBG5tuSi9wqK5lZflHoH1DG2PKN/INzGpmt
m+kGQWWn9vlZVt6RJIVE2e3c9tDH5yC3bfLD5xDeCdfp2FM0ZEZTd+hn8F1BwLqbvEuu97dgbF/N
K7W4C2HV/m19pSN0vkQcv5oCGOp6zQHDQ5f2qwUor4aNADbjRPtdKVIIED9Z3IjKljb0qPbR3FxP
GWicBOWG1m/Sm5AmDd2VoJOf9+yjeyBV5fLqGpr3RHvEV0z0VjNSYy5C8/ycN6g/JrfnQqU2SeAd
+cUfGxDzhkbO/Wnk5bRFCHYdyPq2aKqzz2hY19VimmUmWAhwyhbFzVNaj0NkGAhdTNeJS+TyJfJi
3/NW/KL0HKMtZR52Vf6FAAhzKeu5JZ5dSzA2zN84WOzGqr6vdfpKn27CIuZdyqPjhoZ1DOoLaowm
tUnsljv9V8gfsFlSPO7V/OIEJWzT96ThvXLLIxQqWigxtBZqQgZaX66zKkyyJhQrUIES2XlsN2oz
q9aianROOLVREsDg7vdAi/04qx9X832SaA8TsO950xlHAAoJEBZ0DdIzBUyGm74QALOHBHnyhp8C
BCrDNanglUS5ku6B27fJbIVQaOd3dw6SeTua3iHUgJwFHihDZtBsfiodH4AXTz0pueAlQ1j6oMhd
VLzRqgcHH12qamnZsjjBDf35skrG6kLyUGgrlXXSgPtwFC4+vypShHAJiBsxEZ2YM46pXvyHdPO4
W/R0PWW0Jc3K4wHhiZp5ymtUqZ4ydH5ItLPC9N3219GFJFrbfHtaPQGev1GaQNTbzmX6wlFkMeVt
Ku4HAaHXnFfI4nDksOemVTCWBxb/nSWw4ZpVnyR0MSJunEsCZjT3duLvK5KarCb1y1oupI0hHYan
g3sKZzTQwDl2oIkLZaEj0SYdToClYyMLBjj2QhOyCZWXpA3GnX+Se6gCz1ODqpOYEIRemLfJhrf9
EkI8gHo037wajP4p6qdFQEXljFJZswpxtYva1PcqmKfrKdPeX01nArPU+uvjSolYPDSbFyqCdG7b
LX9WNDtkebOcMcO0Z8NSnqK1cLX6nWpvfKh52otDwi9HJ6eJPgejgRsIqBpQ/q6vvYAbA5FhyAwg
l9O0D0jxBvBaKwpaipifvJEfaqOGTBnikDO7PVyohR2O3DsaX4V0pSqLA6lBDnecYj9hDFhChZe9
jRLCcadkVNNH8tE20qUvqWeXs+5+cYUfcnb1PSlZmD2NeOGpcQXUl/Jlw945/IXT
=sEmG
-----END PGP PUBLIC KEY BLOCK-----

Reply:
![image](https://user-images.githubusercontent.com/84606625/181163731-056a6873-64c5-4c4f-a06b-8e8f4ccfb536.jpeg)

Reply:
![image](https://user-images.githubusercontent.com/84606625/181163796-7fb0c3e9-bef4-4e2e-9091-2a4c5e8edc45.jpeg)